crates/chain/src/tx_graph.rs

```diff
@@ -319,6 +319,30 @@ impl<A> TxGraph<A> {
             .map(|(outpoint, spends)| (outpoint.vout, spends))
     }
 
+    /// Creates an iterator that filters and maps ancestor transactions.
+    ///
+    /// The iterator starts with the ancestors of the supplied `tx` (ancestor transactions of `tx`
+    /// are transactions spent by `tx`). The supplied transaction is excluded from the iterator.
+    ///
+    /// The supplied closure takes in two inputs `(depth, ancestor_tx)`:
+    ///
+    /// * `depth` is the distance between the starting `Transaction` and the `ancestor_tx`. I.e., if
+    ///    the `Transaction` is spending an output of the `ancestor_tx`; the `depth` will be 1.
```

Comment:
```suggestion
    ///    the `Transaction` is spending an output of the `ancestor_tx` then `depth` will be 1.
```
^Grammatical error also exists in the `walk_descendants` function (`;` shouldn't be used to join conditional clauses; it's for independent clauses).